sdk/eventhub/Microsoft.Azure.EventHubs/tests/Client/MiscTests.cs

```diff
@@ -125,5 +125,23 @@ await using (var scope = await EventHubScope.CreateAsync(1))
                 }
             }
         }
+        [Fact]
+        [LiveTest]
+        [DisplayTestMethodName]
+        public async Task ClosingEventHubClientClosesChildEntities()
```

Comment:
nit:  This test is a bit chunky, arguably having responsibility for more than one scenario.   Maybe consider breaking it up into distinct cases:

- Event Hub client marks itself when closed
- Event Hub client  marks a child sender
- Event Hub client marks a child receiver